doc/liblouis.texi

```diff
@@ -1030,30 +1051,30 @@ considered a phrase.
 lenemphphrase underline 3
 @end example
 
+With the above rule, a sequence of two emphasized words will not be
+indicated as a phrase, but the words will be indicated individually.
+
 @end table
 
-@node Symbol
-@subsubsection Symbol
-UEB has a concept of symbols that need special indication. When the
-translator detects an emphasis sequence that needs to be indicated
-with the rules for a symbol then it will use the dots defined with the
-@opcoderef{emphletter}. To indicate the end of the symbol it will use
-the dots defined in the @opcoderef{endemphword}.
-
-@node Fallback behavior
-@subsection Fallback behavior
-
-Many braille systems either handle emphasis using no contexts or
-otherwise by employing a combination of the letter, word and phrase
-contexts. So if a table defines any opcodes for the letter, word or
-phrase contexts then liblouis will signal an error for opcodes that
-define emphasis with no context. In other words contrary to previous
-versions of liblouis there is no fallback behavior.
-
-As a consequence, there will only be emphasis for a context when the
-table defines it. So for example when defining a braille dot pattern
-for phrases and not for words liblouis will not indicate emphasis on
-words that aren't part of a phrase.
+@node Note about fallback behavior
+@subsection Note about fallback behavior
+
+Contrary to older versions of liblouis there is limited fallback
+behavior. Generally opcodes have a very specific purpose. This is done
+for the sake of simplicity. While it would be possible to support more
+combinations of rules, Liblouis chooses to signal an error when
+certain combinations of opcodes are used (or not used).
+
+For example, when a @code{begemphphrase} rule is defined it is
+required that there is also an @code{endemphphrase}
+definition. @code{begemph} must be combined with @code{endemph}. A
+@code{begemphphrase} rule is only allowed if there is also a
+@code{begemphword}. @code{begemph} and @code{begemphword} are mutually
+exclusive. Etc.
+
+When new requirements for indicating emphasis arise that are not
+supported yet, either more opcode combinations might be enabled, or
+more specific opcodes might be added.
```

Comment:
Is this paragraph needed? I think it is clear that we might support more features in the future. IMHO that goes without saying.

Reply:
Hmm, actually I struggled quite a bit with this section. I wanted it to contain something that explained there is a trade-off between a bigger number of more specific and exclusive opcodes on the one hand, and fewer and more general opcodes that can be combined more freely on the other hand.

Of course it's obvious that we will support more features in the future, but the emphasis lies more on the second part, regarding how we can achieve it.

Reply:
OK, fine with me